src/main/java/com/hwr/hwrzeiterfassung/database/controller/LoginController.java

```diff
@@ -0,0 +1,40 @@
+package com.hwr.hwrzeiterfassung.database.controller;
+
+import com.hwr.hwrzeiterfassung.database.repositorys.LoginRepository;
+import com.hwr.hwrzeiterfassung.database.tables.Login;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.util.Optional;
+
+@Controller
+@RequestMapping("/login")
+public class LoginController {
+    @Autowired
+    private LoginRepository loginRepository;
+
+    @GetMapping("/basicLogin")
```

Comment:
We send a password in this request. I don't think it should be a get request, even if the password will surely be hashed